tests/commands/pootle_runner.py

```diff
@@ -30,7 +30,8 @@ def test_pootle_version(capfd):
 
 
 @pytest.mark.cmd
-def test_pootle_init(capfd):
+@pytest.mark.django_db
+def __test_pootle_init(capfd):
     """pootle init --help"""
     call(['pootle', 'init', '--help'])
```

Comment:
This doesn't write out a ``pootle.conf`` file. So not 100% sure why its needs disabling